File: src/main/java/com/telegram/folobot/mappers/FoloUserMapper.java
package com.telegram.folobot.mappers;

import com.telegram.folobot.domain.FoloUserEntity;
import com.telegram.folobot.dto.FoloUserDto;
import org.springframework.stereotype.Component;

@Component
public class FoloUserMapper {
    public FoloUserDto mapToFoloUserDto(FoloUserEntity entity){
        return new FoloUserDto()
                .setUserId(entity.getUserId())
                .setMainId(entity.getMainId())
                .setName(entity.getName())
                .setTag(entity.getTag());
    }

    public FoloUserEntity mapToFoloUserEntity(FoloUserDto dto){
        return new FoloUserEntity()
                .setUserId(dto.getUserId())
                .setMainId(dto.getMainId())
                .setName(dto.getName())
                .setTag(dto.getTag());
    }
}

File: src/main/kotlin/com/telegram/folobot/mappers/FoloUserMapper.kt
package com.telegram.folobot.mappers

import com.telegram.folobot.domain.FoloUserEntity
import com.telegram.folobot.dto.FoloUserDto
import org.springframework.stereotype.Component

@Component
class FoloUserMapper {
    fun mapToFoloUserDto(entity: FoloUserEntity): FoloUserDto {
        return FoloUserDto(entity.userId, entity.mainId, entity.name , entity.tag)
    }

    fun mapToFoloUserEntity(dto: FoloUserDto): FoloUserEntity {
        return FoloUserEntity(dto.userId, dto.mainId, dto.name, dto.tag)
    }
}